docs/plugins/included_source.js

```diff
@@ -69,11 +72,49 @@ async function getIncludedSources(options) {
     }));
     sourceFiles = sourceFiles.flat().filter(pair => pair.length > 0);
 
-    // finally, write all the source files in the `sourceDir`
+    // finally, write all the source files in the source directory
     return await Promise.all(sourceFiles.map(async ([sourceFile, content]) => {
-        return await outputFile(`${sourceDir}/${sourceFile}`, content);
+        return await outputFile(`${docsDir}/${relativeSourceDir}/${sourceFile}`, content);
     }));
 };
 
 
+/**
```

Comment:
Can we please add a short sentence about what this is doing?

Reply:
ah yeah, totally forgot about this, thanks for the heads up